### PR TITLE
Cleaned up the :Tutor file for refrencing Neovim rather than Vim

### PR DESCRIPTION
**Pull request by RoryNesbitt**
## Why

The :Tutor file makes several refrences to Vim or a vimrc file throughout it and while it makes sense for experienced users it is confusing for someone who is starting Neovim with no Vim experience.

## What

Changed all references to Vim and vimrc to be more accurate to neovim

Added a line in Lesson 0 that tells a user to use `:q` to close the help split, which will also fix #15896

Changed the spacing in the <kbd>hjkl</kbd> explanation. This one is just a pet peeve but it took me forever to not get <kbd>j</kbd> and <kbd>k</kbd> backwards

Added capital keys to all end of line commands since #13268 makes this a consistent feature among all commands

Added a note to lesson 3.1 to show that <kbd>P</kbd> can also be used to put before the cursor

Of course updated the json throughout to keep the line numbers accurate

And lastly a cheeky wee commit to add my name to the  file that it's totally  fine if you wanna remove but would be cool if you didn't

## Ongoing

So there's been a fair bit more added to this PR and I suppose while it's active I should say feel free to comment more things you think should be changed, I'm always open to suggestions

### CODE REVIEW

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:1]: proper capitalization is `Neovim`.

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:23]: maybe "help window" is better? "split" is a technical Vim term that beginners are likely not familiar with

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:49]: why the changed alignment?

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:223]: This is a general question (which I don't know the answer to): is knowing two different ways to do the same thing helpful or confusing at this point? (`d$` teaches two things: the command _and_ the motion, as well as the general structure -- if that instruction were expanded, I'd prefer to add an explanation rather than a different command.)

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:896]: I think we can stop referring to Vi at this point; at the same time we don't need to stress the differences to Vim here. Maybe just "Neovim is a very configurable editor. To set up your personalized configuration, you have to create..."?

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:897]: We only refer to `init.vim` in the following; maybe keep it to that here as well. (Don't introduce complexity _without necessity_.)

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor:49]: Having the j and k slightly offset indicates the order they are on the keyboard without someone less used to it having to stop and check. As I mentioned it took me ages to not get them backwards and I think this slight change will help new users

Comment (clason) [reply runtime/tutor/en/vim-01-beginner.tutor:49]: Ah, okay, I was mostly going by the arrows; having them not aligned tweaked my OCD ;)

Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor:49]: it looks a bit jank in githubs preview but it's better in a terminal. At least with fira it's better

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor]: This one feels like when you were in school in physics and every year starts out "We taught you that x is a fact, well actually that's not true, it's sometimes true but here's the truer true"

That being said I agree with your point and have rectified it the latest commit

Comment (clason) [reply runtime/tutor/en/vim-01-beginner.tutor]: Maybe then say "To set up your configuration, you _can_ create..." so it's not wrong?

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor]: I can't think of a good way to word the explanation. I was thinking just add a note under at the end of the lesson, but I'm not sure what to put

However, I have still removed the "or [DCY]" from those lines

Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor]: As to your inital question here however.. I'm not sure that showing both ways is better but I would say learning D is better in the long run than learning d$, I just don't want to take away from teaching what $ means by itself

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:176]:
```suggestion
 5. If you have quit this tutorial in step 1, restart and move down
```

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:553]:
```suggestion
    substitution is to be done (i.e., `1,3` means from line 1 to line 3, inclusive).
```

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:568]:
```suggestion
NOTE: You can also select the lines you want to substitute first using visual-mode.
```

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:821]:
```suggestion
There are many settings in Neovim that you can configure to suit your needs.
```

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:843]: Not sure which of these terms is more widely understandable; personally, I'm more familiar with "toggle" -- but I'm hardly the target audience for this...

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:913]:
```suggestion
To start using more features you should create an "init.vim" file.
```

Comment (clason) [reply runtime/tutor/en/vim-01-beginner.tutor:913]: maybe "can"?

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:956]:
```suggestion
 5. Create an init.vim startup script to keep your preferred settings.
```

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (clason) [inline runtime/tutor/en/vim-01-beginner.tutor:967]: Why did you remove the references? Is the book really not applicable?

In general, I think it's fine to also recommend Vim resources to very beginners, since the fundamentals are not different. Maybe leave them in but preface them with an explanation? ("For the basics, resources on learning Vim are also helpful since the fundamental editing model is the same, but be aware that some options may be set differently by default. Here's a partial list.")

Feel free to remove resources that are too stale, though. I wouldn't stress the "maintainers of Neovim", either.

**Review by clason — APPROVED** [review]
Almost there :)

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor:843]: This change is to match the inv prefix it is talking about, hoping it will help people remember

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor:967]: I was just making the assumption most people don't bother with these and jump straight to YouTube or Google. I shall update how you suggested

Comment (clason) [reply runtime/tutor/en/vim-01-beginner.tutor:967]: Yeah, which drops them into a sea of bad/outdated/monetized advice ;) That's exactly why a curated list is helpful.

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (Mcculloughj) [reply runtime/tutor/en/vim-01-beginner.tutor]: Thank you

File: runtime/tutor/en/vim-01-beginner.tutor
Comment (RoryNesbitt) [reply runtime/tutor/en/vim-01-beginner.tutor:913]: I don't like can since the previous line is "You can.." so I just removed you should

**Comment by RoryNesbitt** [comment]
@clason Do you want me to go back through and change the commit messages for the linter? Otherwise I would call this done I think

**Comment by clason** [comment]
Yes, if you could squash these commits and use a conventional commit message, that would be great.

**Comment by RoryNesbitt** [comment]
My need to capitalise made that take more tries than it should have

**Comment by RoryNesbitt** [comment]
@clason Should request a review again from you since all checks have passed or are we now just waiting for a second person to approve?

Also apologies for the spam, I don't know the next steps here

**Comment by clason** [comment]
No, all good! Thank you for your contribution; improving documentation is no less useful than fixing bugs and adding features!

If you want to update the other chapters (at your own pace), that would be more than welcome.

**Comment by RoryNesbitt** [comment]
Thank you very much, this will be my one bit of infamy in for future nerds.
Am I write in saying this will be part of 0.7?

**Comment by clason** [comment]
Yes, this will part of the next release (0.7.0, maybe in a month or two, or three).

**Pull request merged**
